ecosystem/sep-0024.md

```diff
@@ -410,20 +410,26 @@ The basic parameters are summarized in the table below.
+`callback` | string | (optional) [`postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) or a URL that the anchor should `POST` a JSON message to when the user successfully completes the interactive flow.
+`on_change_callback` | string | (optional) [`postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) or a URL that the anchor should `POST` a JSON message to when the `status` or `kyc_verified` properties change.
 
 The URL supplied by both callback parameters should receive the full transaction object.
 
 **`callback` details**
 
-If the wallet wants to be notified that the user has completed the anchor's interactive flow (either success or failure), it can add this parameter to the URL. If the user abandons the process, the anchor does not need to report anything to the wallet. If the `callback` value is a URL, the anchor must `POST` to it with a JSON message as the body. 
+If the wallet wants to be notified that the user has completed the anchor's interactive flow (either success or failure), it can add this parameter to the URL. If the user abandons the process, the anchor does not need to report anything to the wallet. If the `callback` value is a URL, the anchor must `POST` to it with a JSON message as the body.
 
-**`postMessage` deprecation**
+**`postMessage` details**
 
-Instead of using `postMessage` callbacks, the wallet should either provide a public-facing URL as the `callback` parameter value or poll the anchor's [`GET /transaction(s)`] endpoint for updates to the transaction's status.
+If provided, the anchor must post a JSON message (either as a JSON-serialized string or a plain javascript object) to `window.opener` via the Javascript [`Window.postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) method. If `window.opener` is undefined, the message must be posted to `window.parent` instead.
 
-The anchor should still support the use of `postMessage` callbacks. If provided, the anchor must post a JSON message (either as a JSON-serialized string or a plain javascript object) to `window.opener` via the Javascript [`Window.postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) method. If `window.opener` is undefined, the message must be posted to `window.parent` instead.
+Because `callback` is used to notify the wallet that the interactive flow is complete, it is common for anchors to make the `postMessage` callback as a result of the user clicking a "Close" button or similar UI element. The wallet can then close the window displaying the anchor's interactive flow using `[Window.close()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/close). This ensures the wallet application receives the transaction's information and the user has a smooth experience.
+
+**Interactive Flow Security Concerns**
```

Comment:
💡 Suggested title name to focus on the security concerns and impacted functionality. I.e. If I'm using postMessage this might trigger me to actually read the security concerns section.
```suggestion
**Security Concerns (`noopener`, `postMessage`)**
```